building/tracks/syllabus/next-exercises.md

```diff
@@ -0,0 +1,27 @@
+# Syllabus - Next Exercises
+
+Having implemented your first concept exercise, the next few exercises should build on that.
+
+We like to have three to five exercises that have no other prerequisites other than the `basics` concept.
+
+Good options are the core primitives or data types of your language.
+E.g. booleans, basic numeric types, strings, dates, and atoms.
+
+Other good options are core concepts such as conditionals, loops, and basic arithmetic.
```

Comment:
This reads like "other good options" refers to "exercises that have no other prerequisites that basics" but that usually does not work for the concepts mentioned here.